xbmc/guilib/GUIFontTTF.h

```diff
@@ -148,12 +192,16 @@ class CGUIFontTTF
                         float maxPixelWidth,
                         bool scrolling);
 
-  float m_height;
+  float m_height{0.0f};
+  float m_aspect{1.0f};
+  float m_lineSpacing{1.0f};
+  bool m_border{false};
   std::string m_strFilename;
```

Comment:
Not changed but is this actually used for anything? I see we had a duplicate below that was slightly different `m_strFileName` which was used by the getter `GetFileName()` but this doesn't seem to be used by anything...

Reply:
On other request becomes them removed.

Yes makes no sense to use as value, if nowhere used.